explosion.py

```diff
@@ -0,0 +1,29 @@
+from constants import *
+
+class Explosion(pygame.sprite.Sprite):
+
+    def __init__(self, center_x, center_y, scale: tuple = (100, 100)) -> None:
+        super().__init__()
+        self.images = []
+        for num in range(5):
+            img_path = os.path.join(IMAGES_DIR, f'explosions/explosion_{str(num)}.png')
+            img = pygame.image.load(f"{img_path}")
+            img = pygame.transform.scale(img, (scale[0] + 30, scale[1] + 30))
+            self.images.append(img)
+        self.index = 0
+        self.image = self.images[self.index]
+        self.rect = self.image.get_rect()
+        self.rect.center = (center_x, center_y)
+        self.explosion_speed = 6
```

Comment:
I think that the speeds in the player and missile class are hard coded like this as well. Maybe we should put these all in constants before the demo

Reply:
Good idea.  That way things can be updated easily.